taskPage1.css

```diff
@@ -0,0 +1,80 @@
+.pending-task {
+    height: 80px;
+}
+
+.task-box{
+    padding: 5px 0;
+    height: 80px;
+    width:90%;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    justify-content: space-between;
+}
+
+.task-name{
+    font-style: normal;
+    font-weight: 400;
+    font-size: 23px;
+    color: #000000;
+}
+
+.task-track {
+    text-align: center;
+    max-width: 80px;
+    background: #FFA8A7;
+    border-radius: 5px;
+    padding: 0 2px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 11px;
+    color: #FFFFFF;
+}
+
+.task-deadline p{
+    line-height: 80px;
+}
+
+.next-button{
+    display: flex;
+    justify-content: flex-end;
+    width: 90%;
+    margin: 0 auto;
+    padding: 25px 0 10px 0;
+}
+
+.next-button p{
+    width:80px;
+    background: #153D80;
+    border-radius: 5px;
+    text-align: center;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+    line-height: 30px;
+}
+
+.next-button p a{
+    width:80px;
+    height:30px;
+    text-decoration: none;
+    color: #FFFFFF;
+}
+
+@media (max-width: 600px) {
+    .pending-task {
+        height: 120px;
+    }
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5693010)